mytools.py

```diff
@@ -740,3 +751,21 @@ def intersection(boxA, boxB):
 
     # compute the area of intersection rectangle
     return dx * dy
+
+
+def predict_stream(model, camera_id):
```

Comment:
I like this method in mytools also. In fact this method is easier to extend to multiple models running on same camera stream. Especially if the multiple models are such that they can be visualized together: e.g. all detection models